scenes/scene.yaml

```diff
@@ -80,10 +88,11 @@ styles:
 sources:
     osm:
         type: MVT
-        url:  https://vector.mapzen.com/osm/all/{z}/{x}/{y}.mvt
+        #url:  https://vector.mapzen.com/osm/all/{z}/{x}/{y}.json
+        mbtiles: nyc.mbtiles
```

Comment:
Well, we might want to cache non-Mapzen sources too. What if I had a web service with vector tiles I wanted cached?

What if we had some sort of field in the metadata saying what the origin is? @pnorman might have some insight.